frontend/src/views/editUser/EditUser.jsx

```diff
@@ -0,0 +1,402 @@
+                    checked={formData.administrator}
+                  />
+                  <RadioGroup
+                    defaultValue="active"
+                    name="govRole"
+                    style={{
+                      gap: 8,
+                      marginTop: 8,
+                    }}
+                    onChange={handleChange}
+                    value={formData.govRole}
+                  >
+                    <FormControlLabel
+                      value="analyst"
+                      control={<Radio />}
+                      style={{
+                        '.MuiFormControlLabel-label': {
+                          fontSize: 16,
+                        },
+                      }}
+                      label="Analyst — can make recommendations on transfers, transactions and compliance reports, manage file submissions and add/edit fuel codes"
+                    />
+                    <FormControlLabel
+                      value="compliance_manager"
+                      control={<Radio />}
+                      label="Compliance Manager — can make recommendations on compliance reports"
+                    />
+                    <FormControlLabel
+                      value="director"
+                      control={<Radio />}
+                      label="Director — can assess compliance reports and approve transactions"
+                    />
+                  </RadioGroup>
+                </>
+              ) : (
+                <Stack spacing={1}>
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Manage Users — can add/edit BCeID users and assign roles"
+                    onChange={handleCheckbox}
+                    name="manageUsers"
+                    checked={formData.manageUsers}
+                  />
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Transfer — can create/save transfers and submit files"
+                    onChange={handleCheckbox}
+                    name="transfer"
+                    checked={formData.transfer}
+                  />
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Compliance Reporting — can create/save compliance reports and submit files"
+                    onChange={handleCheckbox}
+                    name="complianceReporting"
+                    checked={formData.complianceReporting}
+                  />
+                  <FormControlLabel
+                    control={<Checkbox />}
+                    label="Signing Authority — can sign and submit compliance reports to government and transfers to trade partners/government"
+                    onChange={handleCheckbox}
+                    name="signingAuthority"
+                    checked={formData.signingAuthority}
+                  />
+                  <FormControlLabel
```

Comment:
I understand it can be confusing! In the wireframe, it is actually a Radio Button. Clicking on it will uncheck all the checkboxes above.

Please review the wireframe, click on the Radio Button, and then follow the arrows.